src/site/content/en/fast/preload-critical-assets/index.md

```diff
@@ -3,89 +3,129 @@ layout: post
 title: Preload critical assets to improve loading speed
 authors:
   - houssein
+  - mihajlija
 description: |
   As soon as you open any web page, the browser requests an HTML document from a
   server, parses the contents of the HTML file, and submits separate requests
   for any other external references. The critical request chain represents the
   order of resources that are prioritized and fetched by the browser.
 date: 2018-11-05
+updated: 2019-09-17
 codelabs:
   - codelab-preload-critical-assets
   - codelab-preload-web-fonts
 ---
 
-As soon as you open any web page, the browser requests an HTML document from a
-server, parses the contents of the HTML file, and submits separate requests for
-any other external references. The **critical request chain** represents the order of resources that are prioritized and fetched by the browser. Lighthouse identifies assets that are on the third level of this chain as _late-discovered_.
+When you open a web page, the browser requests the HTML document from a server, parses its contents, and submits separate requests for any referenced resources. As a developer, you already know about all the resources your page needs and which of them are the most important. You can use that knowledge to request the critical resources ahead of time and speed up the loading process. This post explains how to achieve that with `<link rel="preload">`.
 
-<img class="w-screenshot" src="./preload-requests.png" alt="Lighthouse's preload key requests audit.">
+## How preloading works
+
+Preloading is best suited for resources typically discovered late by the browser. 
+
+<figure class="w-figure">
+<img src="./network-waterfall-before.png" alt="Screenshot of Chrome DevTools Network panel.">
+<figcaption class="w-figcaption">In this example, Pacifico font is defined in the stylesheet with a <a href="https://developers.google.com/web/fundamentals/performance/optimizing-content-efficiency/webfont-optimization#defining_a_font_family_with_font-face)"><code>@font-face</code></a> rule. The browser loads the font file only after it has finished downloading and parsing the stylesheet.</figcaption>
```

Comment:
Note to @mfriesenhahn & @robdodson. Whenever we see links to developers.google.com/web that's could be a cue of what content we should migrate first.